<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qtpfsgui options</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >:/new/prefix1/images/qtpfsgui.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListWidget" name="OptionListWidget" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="resizeMode" >
        <enum>QListView::Fixed</enum>
       </property>
       <property name="spacing" >
        <number>2</number>
       </property>
       <property name="viewMode" >
        <enum>QListView::ListMode</enum>
       </property>
       <property name="modelColumn" >
        <number>0</number>
       </property>
       <property name="uniformItemSizes" >
        <bool>true</bool>
       </property>
       <property name="currentRow" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>RAW Import</string>
        </property>
        <property name="icon" >
         <iconset resource="../icons.qrc" >:/new/prefix1/images/raw_import.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>HDR Visualization</string>
        </property>
        <property name="icon" >
         <iconset resource="../icons.qrc" >:/new/prefix1/images/appearance.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Tonemapping</string>
        </property>
        <property name="icon" >
         <iconset resource="../icons.qrc" >:/new/prefix1/images/tonemap.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>1</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="page" >
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Raw Import Options&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Qtpfsgui uses &lt;span style=" font-weight:600;">dcraw&lt;/span>'s code to load raw files.&lt;/p>&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Please send me your thoughts in case you think&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">that these dcraw's default parameters are not ok.&lt;/p>&lt;/body>&lt;/html></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3" >
              <property name="title" >
               <string>White balance</string>
              </property>
              <layout class="QVBoxLayout" >
               <property name="margin" >
                <number>2</number>
               </property>
               <property name="spacing" >
                <number>3</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBox_camwb" >
                 <property name="toolTip" >
                  <string>dcraw's -w option &lt;br>default value is &lt;b>true&lt;/b></string>
                 </property>
                 <property name="text" >
                  <string>Use color balance specified by the camera</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_autowb" >
                 <property name="toolTip" >
                  <string>dcraw's -a option&lt;br>default value is &lt;b>false&lt;/b></string>
                 </property>
                 <property name="text" >
                  <string>Automatic color balance</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_4colors" >
                 <property name="toolTip" >
                  <string>dcraw's -f option:&lt;br>Use this if the output shows false 2x2 meshes with VNG or mazes with AHD. &lt;br>default value is &lt;b>false&lt;/b></string>
                 </property>
                 <property name="text" >
                  <string>Interpolate RGB as four colors</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title" >
               <string>Highlights</string>
              </property>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>2</number>
               </property>
               <property name="spacing" >
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="label_7" >
                 <property name="toolTip" >
                  <string>dcraw's -H option&lt;br>read dcraw's man page for explanations &lt;br> default value is &lt;b>0&lt;/b></string>
                 </property>
                 <property name="text" >
                  <string>Highlights recovery mode:</string>
                 </property>
                 <property name="buddy" >
                  <cstring>spinBox_highlights</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_highlights" >
                 <property name="toolTip" >
                  <string>dcraw's -H option&lt;br>read dcraw's man page for explanations &lt;br> default value is &lt;b>0&lt;/b></string>
                 </property>
                 <property name="maximum" >
                  <number>9</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title" >
               <string>Color space</string>
              </property>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>2</number>
               </property>
               <property name="spacing" >
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="label_8" >
                 <property name="text" >
                  <string>Raw conversion output color space:</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_outcolspace" >
                 <property name="currentIndex" >
                  <number>4</number>
                 </property>
                 <property name="maxVisibleItems" >
                  <number>5</number>
                 </property>
                 <item>
                  <property name="text" >
                   <string>Raw color</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>sRGB</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>Adobe RGB</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>Wide Gamut RGB</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>ProPhoto RGB</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title" >
               <string>Quality</string>
              </property>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>2</number>
               </property>
               <property name="spacing" >
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="label_9" >
                 <property name="text" >
                  <string>Conversion quality:</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_quality" >
                 <property name="currentIndex" >
                  <number>2</number>
                 </property>
                 <property name="maxVisibleItems" >
                  <number>3</number>
                 </property>
                 <item>
                  <property name="text" >
                   <string>Bilinear</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>VNG</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>AHD</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2" >
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">HDR Visualization Options&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox" >
              <property name="title" >
               <string>Colors</string>
              </property>
              <layout class="QVBoxLayout" >
               <property name="margin" >
                <number>5</number>
               </property>
               <property name="spacing" >
                <number>8</number>
               </property>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_4" >
                   <property name="text" >
                    <string>Show Negative Numbers as:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer>
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" >
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="negative_color_button" >
                   <property name="text" >
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_5" >
                   <property name="text" >
                    <string>Show nan and +-Inf values as:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer>
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" >
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="infnan_color_button" >
                   <property name="text" >
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>181</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3" >
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Tonemapping Options&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2" >
              <property name="title" >
               <string>Visualization</string>
              </property>
              <layout class="QVBoxLayout" >
               <property name="margin" >
                <number>9</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBox_currentsize" >
                 <property name="toolTip" >
                  <string>if enabled, qtpfsgui will keep the current size of the preview when switching size&lt;br> If disabled, qtpfsgui will switch to the smallest size</string>
                 </property>
                 <property name="layoutDirection" >
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text" >
                  <string>Keep current size when switching tabs</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10" >
                 <property name="text" >
                  <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If enabled, qtpfsgui will keep the current size of the preview when switching size.&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If disabled, qtpfsgui will switch to the smallest size.&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt;">Warning:&lt;/span> some operators require a lot of computation time. Working on a smaller size diminishes the time they require to compute their result.&lt;/p>&lt;/body>&lt;/html></string>
                 </property>
                 <property name="scaledContents" >
                  <bool>true</bool>
                 </property>
                 <property name="wordWrap" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>OptionListWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>617</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
